README.md

```diff
@@ -137,7 +137,7 @@ appium:noSign | Set it to `true` in order to skip application signing. By defaul
 
 Capability Name | Description
 --- | ---
-appium:skipUnlock | Whether to skip the check for lock screen presence (`true`). By default UiAutomator2 driver tries to detect if the device's screen is locked before starting the test and to unlock that (which sometimes might be unstable). Note, that this operation takes some time, so it is highly recommended to set this capability to `true` and disable screen locking on devices under test.
+appium:skipUnlock | Whether to skip the check for lock screen presence (`true`). When set to false then UiAutomator2 driver tries to detect if the device's screen is locked before starting the test and to unlock that (which sometimes might be unstable). Note, that this operation takes some time, so it is highly recommended to set this capability to `true` and disable screen locking on devices under test.
```

Comment:
it still makes senseto describe which behaviour is the default one: `The default driver behaviour is to verify the presence of the screen lock (e.g. `false` value of the capability)`